easybuild/easyconfigs/j/JasPer/JasPer-1.900.1-CrayGNU-17.08.eb

```diff
@@ -0,0 +1,20 @@
+#
+easyblock = 'ConfigureMake'
+
+name = 'JasPer'
+version = '1.900.1'
```

Comment:
=> http://www.ece.uvic.ca/~frodo/jasper/software/jasper-2.0.12.tar.gz ?

Reply:
Done.

Reply:
Installation for version 1.900.1 work, but for version 2.0.12 it fails currently with:
```
omlins@dom101:~/9_soft_install/gromacs/production.git> eb --detect-loaded-modules=unload easybuild/easyconfigs/j/JasPer/JasPer-2.0.12-CrayGNU-17.08.eb -r -f
== temporary log file in case of crash /dev/shm/omlins/easybuild/stage/tmp/eb-4AhzlX/easybuild-B_w7p6.log
== resolving dependencies ...
== processing EasyBuild easyconfig /users/omlins/9_soft_install/gromacs/production.git/easybuild/easyconfigs/j/JasPer/JasPer-2.0.12-CrayGNU-17.08.eb
== building and installing JasPer/2.0.12-CrayGNU-17.08...
== fetching files...
== creating build dir, resetting environment...
== unpacking...
== patching...
== preparing...
== configuring...
== FAILED: Installation ended unsuccessfully (build directory: /dev/shm/omlins/easybuild/stage/build/JasPer/2.0.12/CrayGNU-17.08): build failed (first 300 chars): cmd " ./configure --prefix=/users/omlins/9_soft_install/gromacs/production.git/software/JasPer/2.0.12-CrayGNU-17.08 " exited with exitcode 127 and output:
/bin/bash: ./configure: No such file or directory

== Results of the build can be found in the log file(s) /dev/shm/omlins/easybuild/stage/tmp/eb-4AhzlX/easybuild-JasPer-2.0.12-20170911.175205.OVERZ.log
ERROR: Build of /users/omlins/9_soft_install/gromacs/production.git/easybuild/easyconfigs/j/JasPer/JasPer-2.0.12-CrayGNU-17.08.eb failed (err: 'build failed (first 300 chars): cmd " ./configure --prefix=/users/omlins/9_soft_install/gromacs/production.git/software/JasPer/2.0.12-CrayGNU-17.08 " exited with exitcode 127 and output:\n/bin/bash: ./configure: No such file or directory\n')
```

Reply:
git rm ...

Reply:
Done.